ddtrace/internal/remoteconfig/worker.py

```diff
@@ -48,6 +48,8 @@ def _agent_check(self) -> None:
                 REMOTE_CONFIG_AGENT_ENDPOINT in endpoints or ("/" + REMOTE_CONFIG_AGENT_ENDPOINT) in endpoints
             ):
                 self._state = self._online
+                # Immediately query the agent for the remote config configuration
+                self._online()
```

Comment:
ok, so currently we have:

1. Process start
2. Wait `_remote_config_poll_interval`
3. call `_agent_check`
4. Wait `_remote_config_poll_interval`
5. call `_online`

?

so this change is two folds, remove step 2 and immediately call `_agent_check`, and then also remove step 4 by having `_agent_check` immediately call `_online`?


writing it out seems obvious, just leaving comment in case I got anything wrong.